apps/base-docs/docs/pages/chain/connecting.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Network Information
-slug: /network-information
-description: Documentation about Base Mainnet and Base Testnet. This page covers network information for the Base network, including network names, descriptions, RPC endpoints, chain IDs, currency symbols, and block explorers.
+title: Connecting
```

Comment:
The section header is "Network Information" so I'm using a different name to avoid 
```
- Network Information
    - Network Information
```